ext/gd/tests/bug73869.phpt

```diff
@@ -14,8 +14,10 @@ var_dump(imagecreatefromgd2(__DIR__ . DIRECTORY_SEPARATOR . 'bug73869a.gd2'));
 var_dump(imagecreatefromgd2(__DIR__ . DIRECTORY_SEPARATOR . 'bug73869b.gd2'));
 ?>
 --EXPECTF--
+%A
```

Comment:
does it occur only for 64 bits ?

Reply:
No, this also can happen on 32bit platforms. The reason is that two 16bit words (unsigned) are read, and the multiplication can overflow `signed int` (this would be particular bad, since GD2 files are platform specifc, and so on little-endian, even a valid big-endian GD2 might cause overflow, and vice versa).

The `%A` placeholders are there to cater to system libgd prior to 2.3.0 (I assume these are still supported), because the additional warnings are not supposed to be triggered there.

Reply:
gotcha !